app/bundles/EmailBundle/Tests/Helper/MailHelperTest.php

```diff
@@ -250,64 +254,30 @@ public function testValidateValidEmails()
 
         foreach ($addresses as $address) {
             // will throw InvalidEmailException if it will find the address invalid
+            /** @phpstan-ignore-next-line */
```

Comment:
Id discourage adding ignore lines whenever possible. What error was phpstan throwing?

Reply:
@diaboloshogunate I am getting phpstan error here because use of deprecated method and class and fixing this would be a big task and it will take too much time and code changes.